pkg/pgmodel/ingestor/dispatcher.go

```diff
@@ -45,9 +45,10 @@ type pgxDispatcher struct {
 	copierReadRequestCh    chan<- readRequest
 	seriesEpochRefresh     *time.Ticker
 	doneChannel            chan struct{}
-	doneWG                 sync.WaitGroup
-	labelArrayOID          uint32
 	closed                 *uber_atomic.Bool
+	metricBatchers         sync.WaitGroup
```

Comment:
I am totally confused here. Can you please explain why do we need to change batcher and disaptcher for alerting e2e tests? I'm quite against this change especially as part of this PR which imho should have nothing to do with how we do batching...